pennylane/ops/qubit/__init__.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This module contains the discrete-variable quantum operations.
+
+The operations are divided into six different files:
+
+* ``arithmetic_ops.py``: Operations that perform arithmetic on the states
+* ``matrix_ops.py``: Generalized operations that accept a matrix parameter,
+  either unitary or hermitian depending
+* ``non_parameteric_ops.py``: All operations with no parameters
+* ``observables.py``: Qubit observables excluding the Pauli gates, which are
+  located in ``non_parameteric_ops.py`` instead.
+* ``parameteric_ops.py``: Core parameteric operations that don't fall into
+  any of the other categories
+* ``qchem_ops.py``: Operations for quantum chemistry applications
+* ``state_preparation.py``: Operations that initialize the state
+
+
+"""
```

Comment:
Just removing the empty lines.
```suggestion
* ``state_preparation.py``: Operations that initialize the state
"""
```